server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/SnapshotInProgressAllocationDecider.java

```diff
@@ -51,45 +48,41 @@ public Decision canForceAllocateDuringReplace(ShardRouting shardRouting, Routing
         return canAllocate(shardRouting, node, allocation);
     }
 
+    private static final Decision YES_NO_SNAPSHOTS_RUNNING = Decision.single(Decision.Type.YES, NAME, "no snapshots are currently running");
```

Comment:
Consider renaming this to `YES_NOT_RUNNING`. It was originally confused with `YES_NO` part